README.md

```diff
@@ -1,2 +1,307 @@
-# woltk
-Web of Life Toolkit
+# Woltka
+
+**Woltka** (Web of Life Toolkit App), is a bioinformatics package for shotgun metagenome data analysis. It takes full advantage of, and it not limited by, the [WoL](https://biocore.github.io/wol/) reference phylogeny. It bridges first-pass sequence aligners with advanced analytical platforms (such as QIIME 2). Highlights of this program include:
+
+- gOTU: fine-grain community ecology.
+- Tree-based, rank-free classification.
+- Combined taxonomic & functional analysis.
+
+Woltka ships with a **QIIME 2 plugin**. [See here for instructions](woltka/q2).
+
+## Contents
+
+- [Installation](#installation)
+- [Example usage](#example-usage)
+- [gOTU analysis](#gotu-analysis)
+- [Input files](#input-files)
+- [Tree-based classification](#tree-based-classification)
+- [Combined taxonomic & functional analyses](#combined-taxonomic--functional-analyses)
+
+
+## Installation
+
+Requirement: Python 3.6 or above.
+
+```bash
+pip install git+https://github.com/qiyunzhu/woltka.git@dev
```

Comment:
You will want to make this change at some point before merging. Probably better not to make this change until everything else is ready to merge.

```suggestion
pip install git+https://github.com/qiyunzhu/woltka.git
```

Reply:
Yes I will!